templates/components/page/contact-us-form.html

```diff
@@ -47,33 +47,17 @@
         {{/if}}
     </div>
 
-    {{#if forms.contact.recaptcha.enabled}}
-        <div class="panel">
-            <header class="panel-header">
-                <h3 class="panel-title">{{lang 'forms.contact_us.captcha'}}</h3>
-            </header>
-            <div class="panel-body">
-                <script type="text/javascript"
-                        src="https://www.google.com/recaptcha/api/challenge?k={{forms.contact.recaptcha.public_key}}">
-                </script>
-                <noscript>
-                    <iframe src="https://www.google.com/recaptcha/api/noscript?k={{forms.contact.recaptcha.public_key}}" height="300" width="500" frameborder="0"></iframe>
-                    <br>
-                    <textarea name="recaptcha_challenge_field" rows="3" cols="40">
-                    </textarea>
-                    <input type="hidden" name="recaptcha_response_field" value="manual_challenge">
-                </noscript>
-            </div>
-        </div>
-    {{/if}}
-
     <div class="form-field">
         <label class="form-label" for="contact_question">{{lang 'forms.contact_us.question'}}
             <small>{{lang 'common.required' }}</small>
         </label>
         <textarea name="contact_question" id="contact_question" rows="5" cols="50" class="form-input"></textarea>
     </div>
 
+    {{#if forms.contact.recaptcha.enabled}}
```

Comment:
Moved the field down after the question description field since it is a practice that it should be the last element before the submit.